spec/lib/vmdb/settings_spec.rb

```diff
@@ -293,54 +293,32 @@
 
           expect(SettingsChange.count).to eq 7
           expect(miq_server.settings_changes.find_by(:key => "/api/token_ttl")).to be nil
-          expect(Vmdb::Settings.for_resource(miq_server).api.new_key).to eq "new value"
-          expect(Vmdb::Settings.for_resource(second_server).api.token_ttl).to eq server_value
+          expect(described_class.for_resource(miq_server).api.new_key).to eq "new value"
+          expect(described_class.for_resource(second_server).api.token_ttl).to eq server_value
```

Comment:
Same thing here, why are we interested in the second server at all in this spec? Shouldn't we be testing this?:

```ruby
expect(described_class.for_resource(miq_server).api.token_ttl).to eq zone_value
```